operator/v1/types_network.go

```diff
@@ -320,6 +320,9 @@ type OVNKubernetesConfig struct {
 type HybridOverlayConfig struct {
 	// HybridClusterNetwork defines a network space given to nodes on an additional overlay network.
 	HybridClusterNetwork []ClusterNetworkEntry `json:"hybridClusterNetwork"`
+	// HybridOverlayVXLANPort defines the VXLAN port number to be used by the additional overlay network.
+	// +optional
+	HybridOverlayVXLANPort *uint32 `json:"hybridOverlayVXLANPort,omitempty"`
```

Comment:
Why is this an optional field? What does it default to when the port number is not configured? Can you add that as a comment?

Reply:
I made it optional because it has a default value in ovn-kubernetes, it defaults to 4789 and I added a comment to that effect

Reply:
thanks.